lib/bandiera.rb

```diff
@@ -44,6 +44,8 @@ def logger
         @logger.formatter = Logger::StackdriverJsonFormatter.new
       end
 
+      @logger.level = Logger.const_get(ENV.fetch('LOG_LEVEL', 'DEBUG').upcase)
```

Comment:
Do you really want the default to be debug?

Reply:
🤔 now you mention it, it does seem a little silly - will update 👍